### PR TITLE
network: honor M or O flag in RA even if IPv6AcceptRA.DHCPv6Cleint=always

### PR DESCRIPTION
**Pull request by yuwata**
Follow-up for ac24e418d9bc988ecf114c464701b35934948178.

The original motivation of the commit and RFE #15339 is to start dhcpv6
client in managed mode when neither M nor O flag is set in the RA.
But, previously, if the setting is set to "always", then the DHCPv6
client is always started in managed mode even if O flag is set in the
RA. Such the behavior breaks RFC 7084.

Inspired by #17616.

cc @ssahani and @pbasista.

### CODE REVIEW

File: man/systemd.network.xml [hunk 1]
```diff
@@ -2086,9 +2086,11 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
         <varlistentry>
           <term><varname>DHCPv6Client=</varname></term>
           <listitem>
-            <para>Takes a boolean, or the special value <literal>always</literal>. When true (the default), the DHCPv6 client will be started when the
-            RA has the managed or other information flag. If set to <literal>always</literal>, the DHCPv6 client will be started even if there is no
-            managed or other information flag in the RA.</para>
+            <para>Takes a boolean, or the special value <literal>always</literal>. When true or
+            <literal>always</literal>, the DHCPv6 client will be started when the RA has the managed or
+            other information flag. If set to <literal>always</literal>, the DHCPv6 client will be also
```
Comment (pbasista) [inline man/systemd.network.xml]: phrasing:

```diff
- will be also
+ will also be
```

and:

```diff
- flag set in the RA
+ flag is set in the RA
```

**Review by pbasista — COMMENTED** [review]
Looks OK to me. I have only added a few phrasing-related notes.

File: man/systemd.network.xml (hunk 1 above)
Comment (yuwata) [reply man/systemd.network.xml]: @pbasista Thanks. Updated.

**Comment by ssahani** [comment]
LGTM

**Comment by yuwata** [comment]
@pbasista and @ssahani Thank you for your review. Setting the green label. Let's wait for the CIs.

**Pull request merged**